utils.js

```diff
@@ -78,8 +78,9 @@ function is_exec_or_higher(guild_roles, member_roles) {
     if (!exec_role) {
         throw new Error('Exec role not in given guild roles');
     }
-    for (let i = 0; i < member_roles.length; i++) {
-        const current_role = member_roles[i];
+    const key_array = member_roles.keyArray();
+    for (let i = 0; i < key_array.length; i++) {
```

Comment:
I feel like using a map would make this better than iterating through the values yourself